setup.py

```diff
@@ -12,6 +12,7 @@
     "f90nml>=1.1.0",
     "numpy",
     "xarray",
+    "pyFV3 @ git+https://github.com/NOAA-GFDL/PyFV3.git@main",
```

Comment:
Should `pyFV3`  be part of the base reqs ? You actually have a `pyFV3_requirements` down below, which can be installed with `pySHiELD[pyFV3]`

Reply:
Fair point. I was concerned by having imports that aren't instaled by default, but I guess if pyFV3 is only imported in the "update" code it's fine

Reply:
That update code should not exist here though, or at least not with using the `pyFV3` repository. Right now `pySHiELD` is only plug-able with a `pyFV3` dycore state, which arguably is not an issue considering that their isn't a ton of dycore floating around, but architecturally we should abstract that layer away